Compile error: no 'now' in 'time'

I'm getting a compile error from mowl:

```
  Compiling mowl v1.3.0
error[E0432]: unresolved import `time::now`
  --> /home/martin/.cargo/registry/src/github.com-1ecc6299db9ec823/mowl-1.3.0/src/lib.rs:26:5
   |
26 | use time::now;
   |     ^^^^^^^^^ no `now` in the root
...
error: aborting due to previous error

For more information about this error, try `rustc --explain E0432`.
error: could not compile `mowl`
```

For time I am getting `time v0.2.25`, according to `cargo tree`. This is the latest version according to `https://github.com/time-rs/time`. According to [the doco for 0.2.25](https://docs.rs/time/0.2.25/time/struct.Time.html#method.now), this *deprecates* `now()`.

Maybe you could use `std::time::SystemTime::now()`, `std::time::Instant::now()`, or the `chrono` crate's `Local::now()` or `Utc::now()`.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -16,7 +16,7 @@ travis-ci = { repository = "saschagrunert/mowl", branch = "master" }
 appveyor = { repository = "saschagrunert/mowl", branch = "master", service = "github" }
 
 [dependencies]
-failure = "0.1.7"
-log = { version="0.4.8", features=["std"] }
-term = "0.6.1"
-time = "0.1.41"
+failure = "0.1.8"
+log = { version = "0.4.14", features = ["std"] }
+term = "0.7.0"
+time = "0.2.25"

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -19,7 +19,7 @@ use failure::Error;
 use log::{Level, LevelFilter, Log, Metadata, Record};
 use std::io::prelude::*;
 use term::{color::*, StderrTerminal};
-use time::now;
+use time::{Format, OffsetDateTime};
 
 /// Initializes the global logger with a specific `max_log_level`.
 ///
@@ -124,7 +124,11 @@ impl Logger {
         if self.enable_colors {
             t.fg(BRIGHT_BLACK)?;
         }
-        write!(t, "[{}] ", now().rfc3339())?;
+        write!(
+            t,
+            "[{}] ",
+            OffsetDateTime::now_utc().lazy_format(Format::Rfc3339)
+        )?;
         if self.enable_colors {
             t.fg(BRIGHT_BLUE)?;
         }

